engine/lib/breadcrumbs.php

```diff
@@ -1,118 +1,45 @@
 <?php
 /**
  * Breadcrumbs
- * Elgg uses a breadcrumb stack. The page handlers (controllers in MVC terms)
- * push the breadcrumb links onto the stack. @see elgg_push_breadcrumb()
  */
 
 /**
  * Adds a breadcrumb to the breadcrumbs stack.
  *
- * See elgg_get_breadcrumbs() and the navigation/breadcrumbs view.
- *
  * @param string       $text The title to display. During rendering this is HTML encoded.
  * @param false|string $href Optional. The href for the title. During rendering links are
  *                           normalized via elgg_normalize_url().
  *
  * @return void
  * @since 1.8.0
- * @see elgg_get_breadcrumbs()
  */
-function elgg_push_breadcrumb($text, $href = false): void {
-	$breadcrumbs = (array) _elgg_services()->config->breadcrumbs;
-	
-	$breadcrumbs[] = [
+function elgg_push_breadcrumb(string $text, $href = false): void {
```

Comment:
``$href`` could be ``string|bool $href``, yes this would allow ``true`` but better than some random object or array